jbehave-core/src/test/java/org/jbehave/core/parsers/RegexStoryParserBehaviour.java

```diff
@@ -149,8 +149,11 @@ void shouldParseStoryWithGivenStoriesWithAnchorParameters() {
                 wholeStory, storyPath);
         assertThat(story.getPath(), equalTo(storyPath));
         assertThat(story.getGivenStories().getStories().size(), equalTo(1));
-        GivenStory givenStory = story.getGivenStories().getStories().get(0);
+        GivenStories givenStories = story.getGivenStories();
+        assertThat(givenStories.requireParameters(), equalTo(false));
+        GivenStory givenStory = givenStories.getStories().get(0);
         assertThat(givenStory.hasAnchorParameters(), equalTo(true));
+        assertThat(givenStory.hasAnchorExamples(), equalTo(false));
```

Comment:
I think it makes sense to add the positive test as well (or update any existing one if present)